simple_history/management/commands/clean_duplicate_history.py

```diff
@@ -69,7 +69,7 @@ def _process(self, to_process, date_back=None, dry_run=True):
             # have changes (in the given period) but
             # `m_qs.values(model._meta.pk.name).distinct()`
             # is actually slower than looping all and filtering in the code...
-            for o in model.objects.all():
+            for o in model.objects..iterator():
```

Comment:
oops! Looks like you left the extra `.` in there. Fix that and I'll approve and merge

Reply:
Bah. Sorry about that. Removed the typo. Thanks!